packages/cli/cli/src/cli.ts

```diff
@@ -18,6 +18,7 @@ import { CliContext } from "./cli-context/CliContext";
 import { getLatestVersionOfCli } from "./cli-context/upgrade-utils/getLatestVersionOfCli";
 import { addGeneratorToWorkspaces } from "./commands/add-generator/addGeneratorToWorkspaces";
 import { formatWorkspaces } from "./commands/format/formatWorkspaces";
+import { generateFdrForWorkspaces } from "./commands/generate-fdr/generateFdrForWorkspaces";
```

Comment:
@kafkas i think lets call the method `generateFdrApiDefinitionForWorkspaces`. That's the most readable and makes it easy for someone to understand what is being generated